lab-anthony/entry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+require('!!file?name=[name].[ext]!./html/index.html');
+require('./scss/base.scss');
+
+const angular = require('angular');
+
+const cowsay = require('cowsay-browser');
+
+var app = angular.module('cowsayApp', []);
+
+app.controller('CowsayController', ['$scope', function($scope){
```

Comment:
Best practice for controllers is to attach everything at the top, define things below with named functions, and exploit the fact that functions are hoisted. That way, when you open a file, you can immediately see everything that's attached to it. I would have done this like: 


```javascript

function(){
  // Don't make a variable called app if you don't need one 
  angular.module('cowsayApp', [])
    .controller('CowsayController, [
      '$scope', 
      CowsayController,
    ]);

  function CowsayController($scope) {
    // properties
    $scope.cowsayText             = 'blahblah';
    $scope.animalThatsTalking  = 'default';

    // methods
    $scope.say              = say;
    $scope.submitText  = submitText;

    // This function puts the text in the input through the cowsay function
    function say(){
      // Whatever 
    }

    // This submits the form and populates the result with the cowsay output
    function submitText(){
      // Whatever
    }
  }

})();
```